osi_trafficcommand.proto

```diff
@@ -461,8 +480,181 @@ message SpeedAction
 
 }
 
+// \brief Longitudinal Distance Action
+//
+// The action tells the traffic participant to reach a longitudinal distance to 
+// another traffic participant. The longitudinal distance is defined as the distance
+// along the lane. 
+//
+// \note This action is aligned with LongitudinalDistanceAction of OpenSCENARIO 1.0
+// defining the reference traffic participanz and the distance.
+//
+
+message LongitudinalDistanceAction 
+{
+    // Action Header of the message
+    //
+	optional ActionHeader action_header = 1;
+	
+    // Unique Id of the refence traffic participant to which the distance is defined.
+    //
+	optional Identifier target_traffic_participant_id = 2;
+	
+	// The distance to reach along the lane
+	//
+	// Unit: m
+	//
+	optional double distance = 3;
+	
+	// Determine the measurement of distance between TrafficParticipant
+	// True: Lateral distance is measured using the distance between closest bounding box points.
+    // False: Reference point distance is used
+    //
+	optional bool freespace = 4;
+	
+	// Define wheather the traffic participant should only reach the distance once 
+	// or follow.
+	// If set to true the traffic particpant shall approach the reference participant
+	// and follow woth the distance specified until the action is aborted.
+	// If set to false the action end as soon as the traffic participant has reached
+	// the 
+	optional bool follow = 5;
+	
+	// Parameter that assings either unlimited dynamics (if omitted) 
+    // or limited maxAcceleration/maxDeceleration/maxSpeed to the action.
+    //
+	optional DynamicConstraints dynamic_constraints = 6;
+}
+
+
+// \brief Lateral Distance Action 
+//
+// The action tells the traffic participant to reach a longitudinal distance to 
+// another traffic participant. The longitudinal distance is defined as the distance
+// along the lane. 
+//
+// \note This action is aligned with LateralDistanceAction of OpenSCENARIO 1.0
+// defining the reference traffic participanz and the distance.
+//
+message LateralDistanceAction
+{
+    // Action Header of the message
+    //
+	optional ActionHeader action_header = 1;
+	
+    // Id of the reference entity the lateral distance shall be kept to.
+    //
+	optional Identifier target_traffic_participant_id = 2;
+	
+    // Lateral distance value. Unit: m; Range: [0...inf]
+    //
+	optional double distance = 3;
+
+	// True: Lateral distance is measured using the distance between closest bounding box points.
```

Comment:
@samnichols-cpc Do you think the bounding box for this distance definition should cover the whole object including side mirrors or not? 
The bounding box definition is discussed in https://github.com/OpenSimulationInterface/open-simulation-interface/issues/444. I think it will be clarified for V3.3 and might change in the long term. From the standpoint of this message we might have a preference.

Reply:
@clemenshabedank  I would argue that the side mirrors should be included in the bounding box due to the fact that if the side mirrors were to collide in a simulation i would argue that should be considered a collision and not ignored. I would argue that either way, the user is going to want the OSI standard to match what THEIR bounding box is, so the decision shouldn't make a difference as either way it wont make everyone happy, I think as long as we have the option to change the distance then everyone should be able to use the Long/Lat DistanceActions

Reply:
Hello @samnichols-cpc, I would argue the same as you that the side mirrors shall probably be included for most usecases (e.g. setting a collision-critical distance). Yet, the current definition excludes the side mirrors, and the decision in the Harmonization with OpenX group seems to leave it like this for V3.3 to ensure backwards compatibility (see https://github.com/OpenSimulationInterface/open-simulation-interface/pull/461). But it is planned to make an option possible to include the side mirrors in V4.0 together with reworking the whole bounding box concept. 

Therefore, in the WP11 meeting yesterday we discussed it is probably best to go with the central definition, which will unfortunately not make the inclusion of side mirrors possible before V4.0.  Would that be fine for you?

Reply:
Hello @clemenshabedank, Yes this is fine by me as long as it is clear they are not included in the bounding box. Thanks for the confirmation.